monai/networks/nets/ahnet.py

```diff
@@ -272,53 +320,67 @@ class AHNet(nn.Module):
     Anisotropic Hybrid Network (AH-Net).
     The code is adapted from lsqshr's original version:
     https://github.com/lsqshr/AH-Net/blob/master/net3d.py
-    In order to use pretrained weights from 2D FCN/MCFCN, please call the copy_from function.
-    In the default settings (same as the author's official code), the input size of the first two dimensions should be divided
+    The model supports 2D or 3D inputs, as for the original 3D version, in order to use pretrained weights 
```

Comment:
the network is designed for anisotropic kernels, technically the 2D case should use kernels such as 3x1. Using 3x3 conv. kernels lose the key feature of the anisotropic design. This will cause confusions. please consider clarifying this in the docstring, or modify the implementation.

Reply:
I see, let me use 3*1 instead.